Heals e-book-backend-ldap.c:826:3: error: implicit declaration of function 'e_get_user_data_dir'
--- evolution-data-server-2.32.3/addressbook/backends/ldap/e-book-backend-ldap.c.~1~	2015-01-11 12:54:13.144149728 +0300
+++ evolution-data-server-2.32.3/addressbook/backends/ldap/e-book-backend-ldap.c	2015-01-11 12:55:36.137322721 +0300
@@ -125,6 +125,7 @@
 
 #include <glib/gi18n-lib.h>
 #include "libedataserver/e-sexp.h"
+#include "libedataserver/e-data-server-util.h"
 #include <libebook/e-contact.h>
 
 #include <libedata-book/e-book-backend-sexp.h>
